### ISSUE TITLE
cMAD question

### ISSUE THREAD

**Issue by carolinehey** [aeeckhou/shallowHRD#19]
I have a question about the cMAD that i hope you can anwser. I the shallowHRD_hg19_1.13_QDNAseq_no_chrX.R code i found this snippet:
![image](https://github.com/aeeckhou/shallowHRD/assets/98043249/1726537f-5b20-4cff-8b9c-2b57820d8f48)
However, further up in the script i see thresholds matching what is specified in the Bioinformatic report:
![image](https://github.com/aeeckhou/shallowHRD/assets/98043249/ea8315a3-d7dc-44df-a5fa-c4bf9af76e66)

Are these cMAD/QC_MAD_point_corrected supposed to be the same, and classified as bad >0.50 or 0.05 ? 
Finally, there is also a threshold cMAD<0.14 is good. 

I am a bit confused and hope you can clarify this.

Kind regards,
Caroline

**Comment by aeeckhou** [comment]
Hello Caroline,

I have included in the very beginning the cMAD feature to use it as a mesure of quality. That part of the code is from very old time. The _cMAD_ variable is deprecated (first snipplet) to the _QC_MAD_point_corrected_ calculation. With it, I defined the first rule as previously described in the Bioinformatics report.  

It is the variable _QC_MAD_point_corrected_ that is used in the final table, not the _cMAD_ !
```
a = c("CNA cut-off value", "Number simulations for cut-off", "Number successfull simulations", "MAX2", "cMAD",  "Number LGAs 10Mb", "HRD")
b = c(THR, format(nb_simulations,scientific = F), number_positive, round(MAX2,3), QC_MAD_point_corrected,   number_LGAs, HRD)
```

This criteria was somewhat satisfying at the time but I changed since then the pipeline, relying on random sampling to insure a better CNA cut-off detection. 

For this version 1.13 (not the 2.0), good quality indicators are :
1. The number of succesfull simulations devided by the total number of simulations. < 10 % was associated with bad qualities.
2. The MAX2 value is a good surrogate marker of tumor content. Above 0.16, the tumor content is probably good.  

I would rely more on them. hope I clarified the "cMAD inclusion part" that should not be there.

Is it clear for you ?

Best regards,
Alexandre Eeckhoutte
